src/main/java/com/team01/carsharingapp/service/impl/PaymentServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package com.team01.carsharingapp.service.impl;
+
+import com.team01.carsharingapp.dto.payment.PaymentDto;
+import com.team01.carsharingapp.dto.payment.PaymentRequestDto;
+import com.team01.carsharingapp.mapper.PaymentMapper;
+import com.team01.carsharingapp.model.Car;
+import com.team01.carsharingapp.model.Rental;
+import com.team01.carsharingapp.service.PaymentService;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class PaymentServiceImpl implements PaymentService {
+    private final PaymentMapper paymentMapper;
+    private final Object rentalService;
+
+    @Override
+    public PaymentDto createPayment(PaymentRequestDto requestDto) {
+        //create test entity rental
+        Rental currentRental = new Rental();
+        currentRental.setId(requestDto.rentalId());
+        Car currentCar = new Car();
```

Comment:
You should create car here? Not get from DB?

Reply:
i need this variables to imulate work class, when carService and rentalService will be impled, i change it, but now i must impl business logic